frontend/src/App.js

```diff
@@ -176,6 +207,7 @@ function App() {
           closeDialog={closeDialog}
           postNote={postNoteState}
           patchNote={patchNoteState}
+          onChangeColor={onChangeColor}
```

Comment:
I believe this line should be under the return statement of each Note component (Line 175)

Reply:
you are right, fixing...